site/it/community/contribute/code.md

```diff
@@ -0,0 +1,136 @@
+
+Per evitare attività duplicate, prima di cominciare a lavorare su una funzionalità non banale, cortesemente consulta [RFCs correnti](https://github.com/tensorflow/community/tree/master/rfcs) e contatta gli sviluppatori nei forum TensorFlow. Talvolta siamo selettivi quando dobbiamo decidere se aggiungere una funzionalità, ed il modo migliore di contribuire ed aiutare il progetto è lavorare su problemi conosciuti. 
+
+## Temi per il nuovo contributore
+
+I nuovi contributori alla ricerca di un primo tema per un contributo al codice TensorFlow, dovrebbero cercare i seguenti tag. Raccomandiamo fortemente che i nuovi contributori affrontino inizialmente progetti “easy” e problematiche "good first issue"; questo aiuta il contributore a familiarizzarsi con il flusso di lavoro della contribuzione, ed agli sviluppatori del core di conoscere il contributore.
+
+- `good first issue`
+- `easy`
+- `contributions welcome`
+
+Se siete interessati a reclutare un team per aiutarvi ad affrontare un problema ampio o una nuova funzionalità, cortemente mandate una email a [gruppo sviluppatori@](https://groups.google.com/a/tensorflow.org/forum/#!forum/developers) e osservate la nostra lista attuale di RFC. 
+
+
+## Revisione del codice
+
+Nuove funzioni, eliminazione di difetti, ed ogni altro cambiamento nel codice base sono soggetti a revisione del codice.
+
+La revisione del codice ricevuto come contributo in un progetto attraverso le "pull request" è una componente fondamentale dello sviluppo di TensorFlow. Incoraggiamo chiunque a cominciare rivedendo codice sottoposto da altri sviluppatori, specialmente in quei casi in cui il codice implementa qualcosa che ti piacerebbe usare.
+
+Qui alcune domande da tenere a mente durante il processo di revisione del codice:
+
+*   *Vogliamo questa cosa in TensorFlow?* Potrebbe piacere usarla? Come utente TensorFlow, ti piacerebbe questa modifica e la useresti? Si tratta di un cambiamento nell'ambito di TensorFlow? Il costo di manutenzione di questa nuova funzione sarebbe maggiore dei suoi benefici?
+*   *Il codice è consistente con le API di TensorFlow?* Le funzioni pubbliche, le classi, ed i parametri sono nominati in modo appropriato e progettati intuitivamente?
+*   *E' inclusa la documentazione?* Le funzioni pubbliche, le classi, i tipi di ritorno e gli attributi memorizzati sono nominati in accordo con le convenzioni di TensorFlow e sono documentati chiaramente? La nuova funzionalità è descritta nella documentazione di TensorFlow ed illustrata, ovunque possibile, con esempi? La documentazione si legge in modo appropriato?
+
+*   *Il codice è leggibile da una persona?* Si evitano ridondanze? I nomi delle variabili dovrebbero essere migliorati per aumentare la chiarezza o la consistenza? Dovrebbero essere aggiunti commenti? Ci sono commenti inutili o estranei che potrebbero essere rimossi?
```

Comment:
we could use also `è facilmente leggibile`

Reply:
If you think is better, I agree.